shared-bindings/wifi/Radio.c

```diff
@@ -283,7 +288,11 @@ STATIC mp_obj_t wifi_radio_start_ap(size_t n_args, const mp_obj_t *pos_args, mp_
         authmode = 1;
     }
 
-    common_hal_wifi_radio_start_ap(self, ssid.buf, ssid.len, password.buf, password.len, args[ARG_channel].u_int, authmode);
+    if (args[ARG_max_connections].u_int < 0 || args[ARG_max_connections].u_int > 10) {
```

Comment:
This is an implementation restriction so please put it in common-hal. shared-bindings checks should apply to all possible implementations.

Reply:
Makes sense. Fixed, and re-tested.